apax/utils/data.py

```diff
@@ -21,28 +21,7 @@ def make_minimal_input():
 
 
 def load_data(data_path):
-    """Non ASE compatible parameters have to be saved in an exta file that has the same
-    name as the datapath but with the extension `_labels.npz`.
-
-    Example
-    -------
-    example for the npz-file::
-
-
-        dipole = np.random.rand(3, 1)
-        charge = np.random.rand(3, 2)
-        mat = np.random.rand(3, 1)
-        shape = ['ragged', 'ragged', 'fixed']
-
-        np.savez(
-            "data_path_labels.npz",
-            dipole=dipole,
-            charge=charge,
-            mat=mat,
-            shape=shape,
-        )
-
-    shape has to be in the same order than the parameters
+    """
```

Comment:
What happend here?

Reply:
the docstring referenced an old unused method where additional data was loaded from an npz file.